packages/packages/recently-edited/src/components/top-bar/posts-list.tsx

```diff
@@ -0,0 +1,32 @@
+import {
+	MenuItem,
+	ListSubheader,
+} from '@elementor/ui';
+
+import ChipByDocType, { Props } from '../top-bar/chip-doc-type';
+import { Post } from '../../utils/fetch-posts';
+
+export type RecentPostsProps = {
+	recentPosts: Post[];
+};
+
+export default function PostsList( { recentPosts }: RecentPostsProps ) {
+	return (
+		<>
+			<ListSubheader sx={ { fontSize: 12, fontStyle: 'italic', pl: 4 } } component="div" id="nested-list-subheader">
+				Recent
+			</ListSubheader>
+
+			{ recentPosts.map( ( { title, edit_url: editUrl, type, id } ) => (
```

Comment:
Instead of changing the variable name to "editUrl" try to do it in the data normalization in the new custom hook.